browser/components/shared/dialogue/Dialogue.js

```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components";
+import Title from '../title/Title';
+import CloseButton from '../button/CloseButton';
+import PropTypes from 'prop-types';
+import Grid from '@material-ui/core/Grid';
+
+function Dialogue(props) {
+    return (
+        <Root>
+            <div className="modal-header">
```

Comment:
Where is the `modal-header` style defined?
Styling should follow the component except for generic themed style properties